cluster/images/etcd/Makefile

```diff
@@ -83,7 +83,7 @@ endif
 # This option is for running docker manifest command
 export DOCKER_CLI_EXPERIMENTAL := enabled
 # golang version should match the golang version of the official build from https://github.com/etcd-io/etcd/releases.
-GOLANG_VERSION?=1.16.15
+GOLANG_VERSION?=1.17.13
```

Comment:
This version will be overwritten by the image `gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20210917-12df099d55`, which sets the environment variable to `1.16.2`:
```
> podman run -it gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20210917-12df099d55
bash-5.1# echo $GOLANG_VERSION
1.16.2
```

Reply:
ok forcing the `GOLANG_VERSION` explicitly to use

Reply:
ok forcing the `GOLANG_VERSION` explicitly to use